<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="searchLabel">
        <property name="text">
         <string>Search: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="searchLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="searchButton">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="gematriaLabel">
        <property name="text">
         <string>Gematria: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="gematriaNumberLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListView" name="listView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Gematria_String"/>
    <addaction name="actionInsert_Gematria_List"/>
    <addaction name="separator"/>
    <addaction name="actionQuery_All"/>
    <addaction name="actionSearch_By_Number"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_Gematria_String">
   <property name="text">
    <string>New Gematria String</string>
   </property>
  </action>
  <action name="actionQuery_All">
   <property name="text">
    <string>Query All</string>
   </property>
  </action>
  <action name="actionInsert_Gematria_List">
   <property name="text">
    <string>Insert Gematria List</string>
   </property>
  </action>
  <action name="actionSearch_By_Number">
   <property name="text">
    <string>Search By Number</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
